pkgs/tools/virtualization/google-guest-configs/default.nix

```diff
@@ -23,9 +23,7 @@ stdenv.mkDerivation rec {
     substitute ${./fix-paths.patch} fix-paths.patch \
       --subst-var out \
       --subst-var-by nvme "${nvme-cli}/bin/nvme" \
-      --subst-var-by sh "${stdenv.shell}" \
-      --subst-var-by umount "${util-linux}/bin/umount" \
-      --subst-var-by logger "${util-linux}/bin/logger"
```

Comment:
unclear if we still need `util-linux` as a `binDep` if we aren't patching anything in? probably not, but i guess i need to test this on a GCE VM